JetUtilities/src/L2Creator.cc

```diff
@@ -34,7 +35,7 @@ L2Creator::L2Creator(CommandLine& cl) {
   era        = cl.getValue<string>  ("era");
   l3input    = cl.getValue<string>  ("l3input",    "l3.root");
   output     = cl.getValue<TString> ("output",     "l2.root");
-  outputDir  = cl.getValue<TString> ("outputDir",       "./");
+  outputDir  = cl.getValue<TString> ("outputDir",       "");
```

Comment:
I had some problem...

Le 27/04/2016 02:24, Alexx Perloff a écrit :
>
> In JetUtilities/src/L2Creator.cc 
> <https://github.com/cms-jet/JetMETAnalysis/pull/10#discussion_r61181576>:
>
> > @@ -34,7 +35,7 @@ L2Creator::L2Creator(CommandLine& cl) {
> >    era        = cl.getValue<string>  ("era");
> >    l3input    = cl.getValue<string>  ("l3input",    "l3.root");
> >    output     = cl.getValue<TString> ("output",     "l2.root");
> > -  outputDir  = cl.getValue<TString> ("outputDir",       "./");
> > +  outputDir  = cl.getValue<TString> ("outputDir",       "");
>
> Why did you change the default value?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/cms-jet/JetMETAnalysis/pull/10/files/6c09ed51401443d614ab03918245241673c2ddb0#r61181576>
>